Replace deprecated SconeServicesProperties and GramineServicesProperties to support the teeFrameworkVersion field (#630)
iExecBlockchainComputing/iexec-worker: CHANGELOG.md

## [[NEXT]](https://github.com/iExecBlockchainComputing/iexec-worker/releases/tag/vNEXT) 2025

### New Features

> - Use enclave version of dApp for retrieving pre/post-compute properties via SMS endpoint. (#630)

Reviewer: ```suggestion
- Use TEE framework version of dApp to retrieve pre/post-compute properties via SMS endpoint. (#630)
```
